scripts/unique-kmers.py

```diff
@@ -38,8 +41,16 @@ def get_parser():
 
     :option:`-k`/:option:`--ksize` should be set to the desired k-mer size.
 
-    Output is sent to STDOUT, but a report file can be generated with
-    :option:`-R`/:option:`--report`.
+    Informational output is sent to STDERR, but a report file can be generated
+    with :option:`-R`/:option:`--report`.
+
+    :option:`--stream-out`:option: will write the sequences taken in to STDOUT.
+    This is useful for workflows: count unique kmers in a stream, do digital
+    normalization, and then count unique kmers again.
+
+    :option:`--diagnostics`:option: will provide detailed options for tablesize
```

Comment:
if there isn't a short version then don't add the second `:option:`. Did you run `make doc` and actually examine the output?